modin/experimental/engines/omnisci_on_ray/frame/data.py

```diff
@@ -451,6 +454,32 @@ def fillna(
 
         return new_frame
 
+    def dropna(self, subset, how="any"):
+        how_to_merge = {
+            "any": "AND",
```

Comment:
Shouldn't values for `any` and `all` be swapped?

Reply:
No. Expression in `checks` shows whether we include row at the result or not. So:
```
col1.is_not_null() AND col2.is_not_null() == all(col1, col2) is not None 
Or if we reverse it to make filter which shows us what line to discard:
col1.is_null() OR col2.is_null() == any(col1, col2) is None # so AND == any

col1.is_not_null() OR col2.is_not_null() == any(col1, col2) is not None
Or if we reverse it to make filter which shows us what line to discard:
col1.is_null() AND col2.is_null() == all(col1, col2) is None # so OR == all
```

Reply:
Okay, I see. Thanks!